packages/tldraw/src/lib/shapes/arrow/ArrowShapeUtil.tsx

```diff
@@ -617,7 +617,8 @@ export class ArrowShapeUtil extends ShapeUtil<TLArrowShape> {
 				</SVGContainer>
 				{showArrowLabel && (
 					<TextLabel
-						id={shape.id}
+						id={shape.id + '-label'}
+						shapeId={shape.id}
```

Comment:
We probably don't want to add this one as the `SVGContainer` sibling already sets the id.